hackathon_site/hardware/views.py

```diff
@@ -2,14 +2,15 @@
 
 from django_filters import rest_framework as filters
 from django.db import transaction
-from django.http import HttpResponseServerError
+from django.http import HttpResponseServerError, HttpResponseForbidden
```

Comment:
`HttpResponseForbidden` is also unused now

Reply:
lol why doesn't black remove unused imports 😢